README.md

```diff
@@ -110,74 +88,50 @@ with napari.gui_qt():
-        [
-            [0, 0, 100, 100],
-            [0, 0, 50, 120],
-            [1, 0, 100, 40],
-            [2, 10, 110, 100],
-            [9, 8, 80, 100],
-        ]
-    )
-    viewer.add_points(
-        points, size=[0, 6, 10, 10], face_color='blue', n_dimensional=True
-    )
+You can also draw lines and polygons on an image with a `Shapes` layer. You can then adjust the properties of then shapes and vertices, including their positions and fill and stroke colors. Run `examples/add_shapes.py` to generate and interact with the following example:
 
-```
+![image](resources/screenshot-add-shapes.png)
 
-![image](resources/screenshot-nD-slicing.gif)
+You can also paint pixel-wise labels with a `Labels` layer, which is useful for creating masks for segmentation. Run `examples/add_labels.py` to generate and interact with the following example:
 
-You can draw lines and polygons on an image, including selection and adjustment of shapes and vertices, and control over fill and stroke color. Run `examples/add_shapes.py` to generate and interact with the following example.
+![image](resources/screenshot-add-labels.png)
 
-![image](resources/screenshot-add-shapes.png)
+You can render and quickly browse slices of multi-dimensional arrays in 2D or visualize them in 3D. Run `examples/nD_labels.py` to generate and interact with the following example:
 
-You can also paint pixel-wise labels, useful for creating masks for segmentation, and fill in closed regions using the paint bucket. Run `examples/add_labels.py` to generate and interact with the following example.
+![image](resources/screenshot-nD-slicing.gif)
 
-![image](resources/screenshot-add-labels.png)
+## tutorials
 
-For details checkout our [in-depth tutorials](https://napari.github.io/napari-tutorials/)
+For more details on how to use `napari` checkout our [in-depth tutorials](https://napari.github.io/napari-tutorials/)
 
 ## plans
 
 We're working on several features, including
 
-- support for 3D volumetric rendering
-- support for multiple canvases
+- support for multiple linked canvases
 - a plugin ecosystem for integrating image processing and machine learning tools
 
-See [this issue](https://github.com/napari/napari/issues/301) for some of the features on the roadmap for our `0.2` release. Feel free to add comments or ideas!
+See [this issue](https://github.com/napari/napari/issues/420) for some of the features on the roadmap for our `0.3` release. Feel free to add comments or ideas!
 
 ## contributing
 
 Contributions are encouraged! Please read our [contributing guide](CONTRIBUTING.md) to get started. Given that we're in an early stage, you may want to reach out on [Github Issues](https://github.com/napari/napari/issues) before jumping in.
+
+## help
+We're a community partner on the [imagesc forum](https://forum.image.sc/tags/napari) and all help and support requests should be posted on the forum with the tag `napari`. We look forward to interacting with you there.
```

Comment:
🎊

Reply:
Is this statement accurate or preempting some events that are yet to happen? =)

Reply:
to happen! but imagesc requests posting a message like this before they allow us to be community members